src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/sequencingRuns/SequencingRunDetailsPageIT.java

```diff
@@ -8,45 +8,41 @@
 import ca.corefacility.bioinformatics.irida.ria.integration.AbstractIridaUIITChromeDriver;
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.LoginPage;
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.sequencingRuns.SequencingRunDetailsPage;
-import ca.corefacility.bioinformatics.irida.ria.integration.pages.sequencingRuns.SequencingRunsListPage;
 
 import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.annotation.DatabaseTearDown;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @DatabaseSetup("/ca/corefacility/bioinformatics/irida/ria/web/sequencingRuns/SequencingRunsPagesIT.xml")
+@DatabaseTearDown("/ca/corefacility/bioinformatics/irida/test/integration/TableReset.xml")
 public class SequencingRunDetailsPageIT extends AbstractIridaUIITChromeDriver {
 	private SequencingRunDetailsPage page;
 
 	@Override
 	@BeforeEach
 	public void setUpTest() {
-		LoginPage.loginAsManager(driver());
+		LoginPage.loginAsAdmin(driver());
 		page = new SequencingRunDetailsPage(driver());
 		page.getDetailsPage(1L);
 	}
 
 	@Test
 	public void testGetDetails() {
 		Map<String, String> runDetails = page.getRunDetails();
-		assertEquals(runDetails.get("Description"), "A cool run");
-		assertEquals(runDetails.get("Workflow"), "test workflow");
+		assertEquals(runDetails.get("Description"), "A cool run", "Description should exist.");
+		assertEquals(runDetails.get("Workflow"), "test workflow", "Workflow should exist.");
```

Comment:
Can you update these asserts, the first param is supposed to be the expected value and the second param is the actual value.

Reply:
Please see 63bdb53f0359fb8f3615a8b435f2eaa9ced71509.